packages/amplify-graphql-transformer/API.md

```diff
@@ -25,7 +25,7 @@ import { UserDefinedSlot } from '@aws-amplify/graphql-transformer-core';
 export const constructTransform: (config: TransformConfig) => GraphQLTransform;
 
 // @public (undocumented)
-export const constructTransformerChain: (options?: TransformerFactoryArgs) => TransformerPluginProvider[];
+export const constructTransformerChain: (options?: TransformerFactoryArgs, allowGen1Patterns?: boolean) => TransformerPluginProvider[];
```

Comment:
Can we put `allowGen1Patterns` into `TransformerFactoryArgs` rather than tacking on a dangling argument?